src/ObjectPool/src/ObjectPool.cs

```diff
@@ -19,6 +19,7 @@ public abstract class ObjectPool<T> where T : class
     /// Return an object to the pool.
     /// </summary>
     /// <param name="obj">The object to add to the pool.</param>
+    /// <exception cref="System.ArgumentNullException">when <paramref name="obj"/> is null.</exception> 
```

Comment:
```suggestion
    /// <exception cref="System.ArgumentNullException"><paramref name="obj"/> is <see langword="null"/>.</exception> 
```
Matches docs like this [elsewhere](https://github.com/search?q=repo%3Adotnet%2Fruntime%20%3Cexception%20cref%3D%22ArgumentNullException%22%3E&type=code).